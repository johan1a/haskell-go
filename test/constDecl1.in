const x int = 2