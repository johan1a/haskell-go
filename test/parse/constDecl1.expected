SourceFile (Package "main") [TopLevelDecl1 (ConstDecl [IdDecl "x"] (TypeName (TypeNameIdentifier "int")) [Num 2])]
